### PR TITLE
0.1.0 release

### PR DESCRIPTION
**Pull request by kevin49999**
- added a CHANGELOG
- package is now available on pub.dev
- added installation section to README
- removed deprecated `author` section from `pubspec.yaml`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # optimizely_plugin
 
+[![pub package](https://img.shields.io/pub/v/optimizely_plugin.svg)](https://pub.dev/packages/optimizely_plugin) [![pub points](https://badges.bar/optimizely_plugin/pub%20points)](https://pub.dev/packages/optimizely_plugin/score) [![likes](https://badges.bar/optimizely_plugin/likes)](https://pub.dev/packages/optimizely_plugin/score) [![popularity](https://badges.bar/optimizely_plugin/popularity)](https://pub.dev/packages/optimizely_plugin/score)
+
```
Comment (longhorn499) [inline README.md:4]: this score is pretty cool: https://pub.dev/packages/optimizely_plugin/score, more things to add 👍

File: pubspec.yaml [hunk 2]
```diff
@@ -1,7 +1,6 @@
 name: optimizely_plugin
 description: Flutter plugin for Optimizely SDK
-version: 0.0.1
-author: Policygenius Engineering
```
Comment (maxnovak) [inline pubspec.yaml:4]: Why the author removal?

Comment (maxnovak) [reply pubspec.yaml:4]: More asking to make sure we don't have something to replace the field with to make sure we keep that #branding

Comment (maxnovak) [reply pubspec.yaml:4]: Also we should delete the commented out stuff we don't need in this file.

Comment (longhorn499) [reply pubspec.yaml:4]: Yep it's been deprecated and replaced with the verified publisher that we need to setup: https://dart.dev/tools/pub/pubspec#authorauthors

File: README.md (hunk 1 above)
Comment (jsocol) [reply README.md:4]: that's neat!

**Review by jsocol — APPROVED** [review]
this looks good to me!

**Pull request merged**